package Chain_of_Responsibility;

import org.junit.Test;

/**
 * Created by dylanb on 16/03/2018.
 */
public class ChainTest {
    @Test
    public void setNextChain() throws Exception {
    }

    @Test
    public void calculate() throws Exception {
    }

}